src/commands/download_folder.ts

```diff
@@ -5,18 +5,31 @@ import { CLIAction } from '../CLICommand/action';
 import {
 	DrivePrivacyParameters,
 	FolderIdParameter,
-	LocalPathParameter,
+	LocalFilePathDownloadParameter,
 	MaxDepthParameter
 } from '../parameter_declarations';
+import { getOutputFolderPathAndName } from '../utils';
+import { join as joinPath } from 'path';
 
 new CLICommand({
 	name: 'download-folder',
-	parameters: [FolderIdParameter, LocalPathParameter, MaxDepthParameter, ...DrivePrivacyParameters],
+	parameters: [
+		FolderIdParameter,
+		{
+			name: LocalFilePathDownloadParameter,
+			description:
+				'(OPTIONAL) the path on the local filesystem where the folder should be downloaded. Defaults to current working directory'
```

Comment:
Temporary overriding the description until we can multiple parameters witht the same name

Reply:
```suggestion
			description:
				'(OPTIONAL) the path on the local filesystem where the folder should be created and into which its contents are then downloaded. By default, the folder is created in the current working directory.'
```